Rakefile

```diff
@@ -0,0 +1,6 @@
+desc 'Generate ruby files from protocol buffers'
+task :gen do
```

Comment:
nit: `gen:proto` ? Also consider using `Makefile` instead in case we need to expose `Rakefile` to the users of the gem

Reply:
`Makefile` sounds good, will fix.